calcom/cal.com
# Zapier Trigger: After Meeting ends

## What does this PR do?

Adds necessary code for new Zapier trigger: After meeting ends 

Fixes #3357

 Loom Video: https://www.loom.com/share/837a7e508af74d6ca4509a1c2460bb2c

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

- Setup Zapier for local environment like described here:  [Zapier Setup](https://github.com/calcom/cal.com/tree/main/packages/app-store/zapier )
   - You don’t need to set up all triggers only set up the new trigger ‘Meeting ended’
- Install Zapier in App Store and copy API key
- Create a zap (with sharing link) that uses the new trigger
- Turn on Zap 
- Check DB if new webhook with ‘MEETING ENDED’ trigger is here 
- Check DB if all confirmed upcoming bookings have a scheduled job
- Book an event 
- Check DB and see if you have a scheduled Job in the new booking 
- Zap should trigger at the end time of the booked event 

Additionally, you can test: 
- Rescheduling 
- Cancelling 
- Recurring events

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
